### ISSUE TITLE
No useful error returned if you try to issue an identity for a non-existant participant on v0.6 runtime

### ISSUE THREAD

**Issue by davidkel** [hyperledger/composer#544]
On a clean xenial ubuntu install I follow the getting started guide to install the digitial property network and this all works fine. I then try to issue an identity but this fails as follows
```
composer identity issue -n digitalproperty-network -i WebAppAdmin -s DJY27pEnl16d -u user1 -a "net.biz.digitalPropertyNetwork.Person#userid" -p defaultProfile

timed out waiting for transaction to complete
Command failed.
```

**Comment by EdProsser** [comment]
This has some docs impact, so I'll pick this one up.

**Comment by caroline-church** [comment]
@davidkel Is this still an issue or can it be closed?

**Comment by davidkel** [comment]
@caroline-church it is an issue for 0.6 if the participant doesn't exist. It's just a case of whether it can be fixed as it is a problem with 0.6 itself not returning errors. It may be possible to check the participant exists before trying to invoke the chaincode to link the identity with a user.
I haven't looked into whether that is possible and it won't be an issue on v1.0 as errors are returned.

**Comment by sstone1** [comment]
@davidkel this is a widespread problem for HLF v0.6 - errors not being returned by the chaincode. We should not attempt to fix or workaround this problem, so closing. Reopen if you disagree 😉

**Comment by davidkel** [comment]
@sstone1 nope, don't disagree just wanted to confirm that we don't need to try to fix on v0.6. Thanks.
